utils.js

```diff
@@ -36,12 +36,15 @@ exports.checkIfUnsupportedFormat = function (data) {
     extName = extenstionReg.exec(data);
     extName = extName && extName.length ? extName[0] : null;
   } catch (err) {
-    console.log('errStr', errStr);
     throw new Error(err);
   }
 
   return extName === 'svg' || extName === 'gif' ? true : false;
 };
 
 exports.SUPPORTED_FILES_COUNT = 'SUPPORTED_FILES_COUNT';
-exports.IMAGE_REGEXP = new RegExp('https://(stag-images|(eu-)?images).(blz-)?contentstack.(io|com)/v3/assets/');
\ No newline at end of file
+exports.IMAGE_REGEXP = new RegExp('https://(stag-images|(eu-)?images).(blz-)?contentstack.(io|com)/v3/assets/');
```

Comment:
not urgent but we need to keep it as default and able to, override via environment variable, in our next major release